layer/layer_utils.h

```diff
@@ -15,11 +15,47 @@
 #ifndef STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_UTILS_H_
 #define STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_UTILS_H_
 
-#if defined(STADIA_PERFORMANCE_LAYERS_NO_GLOG)
-#include <iostream>
-#define LOG(KIND) std::cerr << "[" #KIND "]\t"
-#else
-#include "base/logging.h"
-#endif
+#include <sstream>
+
+namespace performancelayers {
+enum class LogMessageKind { INFO, WARNING, ERROR };
+
+// Helper class for printing log messages. Exposes a common logging macro,
+// SPL_LOG, independent of the underlying logging library. Sample use:
+//   SPL_LOG(WARNING) << "Cannot load file: " << my_file_path;
+//
+// Note that there is no need to add a newline character at the end -- this is
+// handled by the implementation.
+class MessageLogger {
```

Comment:
nit: should this be in a `layers/log.h` file instead?  To prevent accumulating unrelated stuff in this generic file.

Reply:
Changed this to logging.h. Thanks for the suggestion.